warp/yul/ToCairoVisitor.py

```diff
@@ -300,7 +300,7 @@ def visit_leave(self, _node: ast.Leave) -> str:
 
     def _make_external_function(self, node: ast.FunctionDefinition) -> str:
         params = ", ".join(self.print(x, split=False) for x in node.parameters)
-        returns = ", ".join(self.print(x, split=True) for x in node.return_variables)
+        returns = ", ".join(self.print(x, split=False) for x in node.return_variables)
```

Comment:
I think the split parameter (and code connected to it) can now be removed from `visit_typed_name`

Reply:
Ack